src/Chapter1.hs

```diff
@@ -620,7 +643,13 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = last n + preLast n
+  where
+    last :: Int -> Int
+    last n = mod (abs n) 10
+    preLast :: Int -> Int
+    preLast n = div (mod (abs n) 100) 10
```

Comment:
This is perfect 👌🏼 

A few additional notes, no action needed! 🙂 
First of all, thanks for writing type signatures in the `where` construction! Even though when it maybe obvious when you write the code, it is always more time-consuming when you read it, so it is very great of you to take it into the habit 🤗 
Also, as you use the same `n` in these functions, you don't need to pass it explicitly, as `n` argument from the function declaration above is in scope in here 🙂 
So this maybe just:

```haskell
lastDigit :: Int
lastDigit = mod (abs n) 10
```

(btw, does it look familiar? 😉 )

Also, `last` is the name of a standard function. Haskell allows name shadowing, that's why it works. Though, in practice, it's usually better to avoid name clashes with other functions cause it could be confusing 🙂